http/handler.go

```diff
@@ -27,6 +27,9 @@ const (
 	// not to use request forwarding
 	NoRequestForwardingHeaderName = "X-Vault-No-Request-Forwarding"
 
+	// NoCORS is the name of the header telling Vault not to use CORS handler.
+	NoCORS = "X-Vault-No-CORS"
```

Comment:
It simply could be how I implemented things, but the CLI requests were causing panics if CORS was enabled since the CLI requests don't include the Origin header. I'll take a second look at things. Honestly I wasn't happy with this, because I had the exact same thought you did: why would the cli care?